dashboard/apps/prototype/migrations/0011_merge.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
```

Comment:
If they have run out of sync on the staging database they will need to be done like this otherwise there will be ghost migrations